CollapseLauncher/Classes/Helper/FileUtility.cs

```diff
@@ -36,9 +36,9 @@ public static class FileUtility
         /// <returns>True if a new file is found, and false if it timed out</returns>
         public static async Task<bool> WaitForNewFileAsync(string directoryPath, int timeoutMilliseconds)
         {
-            var tcs     = new TaskCompletionSource<string>();
-            var cts     = new CancellationTokenSource(timeoutMilliseconds);
-            var watcher = new FileSystemWatcher(directoryPath);
+            TaskCompletionSource<string> tcs     = new();
```

Comment:
this seems unnecessary as `var` should be enough

Reply:
Resolved